src/forms.less

```diff
@@ -32,6 +32,12 @@ input, select {
     .input-block {
         width: 100%;
     }
+
+    textarea{
+        max-width: 100%;
+        max-height: 90vh;
+    }
+
     // TODO: make these classes implicit. We can find out what class to apply based on its type
```

Comment:
As @rhyneav said, add a class to remove the resizing dots such as `.no-resize` for example.

Reply:
`.no-size` would be perfect, good call!